src/pages/Environments.js

```diff
@@ -202,23 +202,26 @@ const Environments = () => {
   const disksByProject = _.groupBy('googleProject', disks)
   const appsByProject = _.groupBy('googleProject', apps)
 
-  const getBillingProjectCell = (billingProject, shouldWarn) => {
+  const forAppText = appType => !!appType ? `for ${_.capitalize(appType)}` : ''
+
+  const getWorkspaceCell = (workspace, appType, shouldWarn) => {
     return h(Fragment, [
-      billingProject,
+      workspace,
       shouldWarn && h(TooltipTrigger, {
-        content: 'This billing project has multiple active cloud environments. Only the most recently created one will be used.'
+        content: `This workspace has multiple active cloud environments ${forAppText(appType)}. Only the latest one will be used.`
       }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.warning() } })])
     ])
   }
-  const renderBillingProjectForApps = app => {
-    const shouldWarn = !_.includes(app.status, ['DELETING', 'ERROR', 'PREDELETING']) &&
-      getCurrentApp(app.appType)(appsByProject[app.googleProject]) !== app
-    return getBillingProjectCell(app.labels.saturnWorkspaceNamespace, shouldWarn)
+  const renderWorkspaceForApps = app => {
+    const { status, appType, googleProject, labels: { saturnWorkspaceName } } = app
+    const shouldWarn = !_.includes(status, ['DELETING', 'ERROR', 'PREDELETING']) &&
+      getCurrentApp(appType)(appsByProject[googleProject]) !== app
+    return getWorkspaceCell(saturnWorkspaceName, appType, shouldWarn)
   }
-  const renderBillingProjectForRuntimes = runtime => {
+  const renderWorkspaceForRuntimes = runtime => {
     const shouldWarn = !_.includes(runtime.status, ['Deleting', 'Error']) &&
       getCurrentRuntime(runtimesByProject[runtime.googleProject]) !== runtime
-    return getBillingProjectCell(runtime.labels.saturnWorkspaceNamespace, shouldWarn)
+    return getWorkspaceCell(runtime.labels.saturnWorkspaceName, shouldWarn)
```

Comment:
Here you are not passing appType (which makes sense because it's not an app. But that means your arguments to getWorkspaceCell are incorrect.